components/item-detail/item-detail.tsx

```
@@ -30,43 +28,217 @@ export function ItemDetail({
+                  ></path>
+                </svg>
+                Information
+              </h2>
+              <div className="space-y-4 divide-y divide-gray-200 dark:divide-gray-800">
+                <div className="flex justify-between items-center pb-3">
+                  <span className="text-gray-600 dark:text-gray-300 font-medium">
+                    Publisher
+                  </span>
+                  <span className="flex items-center bg-gray-100 dark:bg-black px-3 py-1 rounded-full border border-gray-200 dark:border-gray-800  ">
+                    <span className="w-4 h-4 rounded-full bg-orange-500 dark:bg-orange-600 mr-2"></span>
+                    <span className="font-medium text-gray-800 dark:text-white">
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)